app/views/edition_changes/show.html.erb

```diff
@@ -17,7 +17,7 @@
         <br/>
         <small>
           Changes between this draft and
-          <%= link_to_if @edition.previously_published_edition, "the previous version",  @edition.previously_published_edition %>.
+          <%= link_to_if @edition.previously_published_edition, "the previous version",  edition_changes_path(@edition.previously_published_edition) %>.
```

Comment:
I have a feeling this might be rather confusing to users. Perhaps it's better to remove the link and keep text-only sentence until we have reasonable edition IDs that might demystify what this link does.